internal/cloud/realm/dependencies.go

```diff
@@ -49,20 +83,43 @@ func (c *client) ImportDependencies(groupID, appID, uploadPath string) error {
 		return err
 	}
 
-	res, resErr := c.do(
-		http.MethodPost,
+	res, err := c.do(
+		http.MethodPut,
```

Comment:
just for my own sanity: i imagine there was a corresponding change for this on the backend?  do you mind linking me the commit?

Reply:
Yes, [this is the commit that adds the new endpoint with job queues](https://github.com/10gen/baas/commit/2a22b089090c18168a841d396d1c240d03c2df5f),.